curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f7800b5174406f2d0df76.md

```diff
@@ -0,0 +1,68 @@
+---
+id: 671f7800b5174406f2d0df76
+title: Task 106
+challengeType: 22
+dashedName: task-106
+---
+
+# --instructions--
+
+This is a review of the entire dialogue you just studied.
+
+Place the following phrases in the correct spot: `design sprint workshop`, `neural networks`, `AI workshop`, `cybersecurity talk`, `fine-tuning the code`, `newest trends in programming`.
```

Comment:
```suggestion
Place the following phrases in the correct spot: 

`design sprint workshop`, `neural networks`, `AI workshop`, `cybersecurity talk`, `fine-tuning the code`, and `newest trends in programming`.
```

I'd suggest keep the blanks simple, with 1 or 2 words. If the word is long, just one should be enough.